### PR TITLE
added trajectory tracking example

### PR DESCRIPTION
**Pull request by yakir12**

### CODE REVIEW

File: example/trajectory_tracking/kalman.jl [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+using GaussianDistributions: ⊕ # independent sum of Gaussian r.v.
+using Statistics
+using GLMakie
+using StaticArrays
+
+function onestep!(p, v)
+    v .+= randn(2)
+    w = randn(2)
+    p .+= v + w
+end
```
Comment (mschauer) [inline example/trajectory_tracking/kalman.jl]:
```suggestion
function onestep(p, v)
    v += @SVector randn(2)
    w = @SVector randn(2)
    p += v + w
    p, v
end
```

File: example/trajectory_tracking/kalman.jl [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+pp = Gaussian(x0, P)
+ps = [pp] # vector of filtered Gaussians
+obs = Point2f0[]
+for i in 1:n
+    global pp
+    # predict
+    pp = F*pp ⊕ Gaussian(zero(x0), Q) #same as Gaussian(Φ*p.μ, Φ*p.Σ*Φ' + Q)
+    # observe
+    onestep!(p, v)
```
Comment (mschauer) [inline example/trajectory_tracking/kalman.jl]:
```suggestion
    p, v = onestep(p, v)
```

File: example/trajectory_tracking/kalman.jl [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    0 0 1 0
+    0 0 0 1
+   ])
+# Q: the covariance of the process noise
+Q = SMatrix{4,4}(Diagonal([0.01, 0.01, 0.1, 0.1]))
+# R: the covariance of the observation noise
+R = SMatrix{2,2,Float64}(I)
+
+
+p = zeros(2) # initial poistion
```
Comment (mschauer) [inline example/trajectory_tracking/kalman.jl]:
```suggestion
p = @SVector zeros(2) # initial poistion
```

File: example/trajectory_tracking/kalman.jl [hunk 4]
```diff
@@ -0,0 +1,71 @@
+using Kalman, GaussianDistributions, LinearAlgebra
+using GaussianDistributions: ⊕ # independent sum of Gaussian r.v.
+using Statistics
+using GLMakie
+using StaticArrays
+
+function onestep(p, v)
+    v += @SVector randn(2)
+    w = @SVector randn(2)
```
Comment (mschauer) [inline example/trajectory_tracking/kalman.jl]:
```suggestion
    w = @SVector 0.1*randn(2)
```

File: example/trajectory_tracking/kalman.jl [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    0 0 0 1
+   ])
+# Q: the covariance of the process noise
+Q = SMatrix{4,4}(Diagonal([0.01, 0.01, 0.1, 0.1]))
+# R: the covariance of the observation noise
+R = SMatrix{2,2,Float64}(I)
+
+
+p = @SVector zeros(2) # initial poistion
+v = ones(2) # initial velocity
```
Comment (mschauer) [inline example/trajectory_tracking/kalman.jl]:
```suggestion
v = @SVector ones(2) # initial velocity
```

File: example/trajectory_tracking/kalman.jl [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+pp = Gaussian(x0, P)
+ps = [pp] # vector of filtered Gaussians
+obs = Point2f0[]
+for i in 1:n
+    global pp
+    # predict
+    pp = F*pp ⊕ Gaussian(zero(x0), Q) #same as Gaussian(Φ*p.μ, Φ*p.Σ*Φ' + Q)
+    # observe
+    p, v = onestep(p, v)
+    xobs = copy(p)
```
Comment (mschauer) [inline example/trajectory_tracking/kalman.jl]:
```suggestion
    xobs = p + @SVector randn(2)
```

**Comment by mschauer** [comment]
Ah, gut, es funktioniert noch nach meinen Vorschlagen?

**Comment by mschauer** [comment]
Ah, great, still working after my amends?

**Comment by yakir12** [comment]
Well, now it is, yes! Looks good to me (give it a try if you like). It's mainly aimed at predicting the next step, rather than overall fit. The general behavior seems stable.

**Comment by mschauer** [comment]
Add a link in the readme!

**Comment by yakir12** [comment]
Done.

**Pull request merged**